readme.md

```diff
@@ -144,6 +147,12 @@ Accepts keywords which ends up as a path in Analytics.
 
 `.track('init', 'backbone')` becomes `/init/backbone`
 
+#### .trackEvent(category, action, [label, value])
```

Comment:
Oops I already changed to options objcet. but @SBoudrias if you prefer other way I can change.